src/auth/auth.service.ts

```diff
@@ -136,6 +135,20 @@ export class AuthService {
     });
   }
 
+  verifyAuthHeader(authHeader: string) {
+    if (authHeader) {
+      const token = authHeader.replace('Bearer ', '');
+      try {
+        const result = this.jwtService.verify(token, {
+          secret: this.configService.get('JWT_SECRET'),
+        });
+        return !!result;
+      } catch {
+        return false;
+      }
+    }
+  }
+
```

Comment:
Does this need to return false if !authHeader?

Reply:
Yes, good catch. It would not allow access anyway but better to be consistent. Thanks, fixing in this current task